synapse/lib/layer.py

```diff
@@ -4208,6 +4237,20 @@ async def getStorNodes(self):
             yield buid, s_msgpack.un(byts)
             await asyncio.sleep(0)
 
+    async def getStorNodesByForm(self, form):
+        '''
+        Yield (buid, sode) tuples for nodes of a given form with props/tags/tagprops/edges/nodedata in this layer.
+        '''
```

Comment:
We should probably check if the form name specified is a real form name and blow up with NoSuchForm if not so that the error is clear

Reply:
I added a check in the Storm API since it felt more natural there